src/backend/allocator/dumb.rs

```diff
@@ -96,7 +96,10 @@ impl AsDmabuf for DumbBuffer {
 
     #[profiling::function]
     fn export(&self) -> Result<Dmabuf, Self::Error> {
-        let fd = self.fd.buffer_to_prime_fd(self.handle.handle(), 0)?;
+        let fd = self.fd.buffer_to_prime_fd(
+            self.handle.handle(),
+            (rustix::fs::OFlags::CLOEXEC | rustix::fs::OFlags::RDWR).bits(),
```

Comment:
small nit, but I think these kind of flags should be moved over to the DRM_* variants:

https://github.com/torvalds/linux/blob/master/include/uapi/drm/drm.h#L851

Reply:
Yeah, unfortunately neither the `drm` nor the `drm-sys` crate expose those.

Reply:
@Drakulix are you fine if I open an issue in `drm` and leave a `FIXME` in the code for now?

Reply:
sure

Reply:
done https://github.com/Smithay/drm-rs/issues/182

Reply:
Given this is now fixed in drm-rs, we might able to solve this in this PR after all? https://github.com/Smithay/smithay/pull/1228

Reply:
Looks like this would require a release of `drm-rs`, the released version does not expose the flags

Reply:
fixed